### ISSUE TITLE
Make the revoked logo a bit more visible in the purchase list

### ISSUE THREAD

**Issue by ExtReMLapin** [gmodstore/gmodstore-issues#190]
![](https://i.imgur.com/OZmZwTe.png)

Would be nice if it was a bit more visible compared to the normal one, so it's easier to spot revoked licenses

What i mean is unlike in V2, you can't spot quickly the revoked ones

A slightly red background (for the entire line) on the revoked ones could do the trick, whatever i'm not a ui guy

**Comment by Marcuzz** [comment]
Yeah. I tried doing a red background but it didn't change the text colours at all, so I tried doing that and it didn't listen. Didn't feel like doing extra css to make it work when I threw this together. Will do at one point though
